### ISSUE TITLE
Bug: Followed "Channels" has same stream duplicated

### ISSUE THREAD

**Issue by ProgrammingByPermutation** [streamlink/streamlink-twitch-gui#797]
## Bug report

- [X] I understand the difference between *Streamlink* and *Streamlink Twitch GUI*.
- [X] This is a bug report and I have read the contribution guidelines.

### Environment details

**Operating system and version:**  Windows 10

**Streamlink Twitch GUI version:**  Stable v1.11.0 and Nightly Build v1.11.0+25.g97717576

**Streamlink version:**  2.1.1

**Configuration details:**  N/A

### Description

Clicking on the Followed "Channels" page I see the same streams listed more than once. I suspect it is grabbing the first page of results a second time when scrolling down after triggering the "Loading" button for the next page of results. I'm seeing most streams listed twice.

Additionally, if I change the sorting from Ascending to Descending I'm able to find streams that do not appear in the reverse order.

I have 144 followed channels, if that helps.

### Expected / Actual behavior

When scrolling, the next page of results on the Followed "Channels" page should be the next page of results and not a duplicate of the first page.

### Reproduction steps

1. Have at least 2 pages of followers (from the GUI's perspective)
2. Navigate to the Followed "Channels" page.
3. Scroll down until the "Loading" button is triggered for the second page of results.
4. Note the duplication of streams and missing streams.

### Log output

```
[2021-04-21T15:57:19.031Z][debug][Application]
Parameters
{
    "_": [],
    "tray": false,
    "hide": false,
    "hidden": false,
    "max": false,
    "maximize": false,
    "maximized": false,
    "min": false,
    "minimize": false,
    "minimized": false,
    "reset-window": false,
    "versioncheck": true,
    "version-check": true,
    "logfile": true,
    "loglevel": "debug",
    "l": "debug",
    "launch": "",
    "goto": ""
}
```

### Additional comments, screenshots, etc.

As you can see here, the stream is listed the first time
![First_Stream_Listing](https://user-images.githubusercontent.com/466408/115585010-40746d00-a299-11eb-979a-5eacb02c168d.png)

And when you scroll down farther it's listed a second time
![Duplicate_Stream_Listing](https://user-images.githubusercontent.com/466408/115585023-436f5d80-a299-11eb-89f1-08fea251fb5b.png)

**Comment by bastimeyer** [comment]
This is a known Twitch API issue which is caused by changes that Twitch has made a while ago on their legacy v5 "kraken" API. They've switched from the `offset`+`limit` request parameters scheme to the `cursor`+`limit` scheme, which was originially added to their new "helix" API.

By changing the legacy API and dropping the old request parameters entirely on this API endpoint, they have introduced a breaking change. The "infinite scroll" mechanism in Streamlink Twitch GUI hasn't been updated to that new scheme yet, as it is not present on any other kraken API endpoint other than the followed channels (AFAIK).

**Comment by bastimeyer** [comment]
https://github.com/twitchdev/issues/issues/237

**Comment by bastimeyer** [comment]
Closing, as this will be solved with the switch to the helix API (#849).
